### ISSUE TITLE
Import from remotes should not be default

### ISSUE THREAD

**Issue by wagenet** [jakubpawlowicz/clean-css#767]
In general, if you're importing remotely, it's because you actually wanted to request that file on load. Overwriting this behavior should be opt-in. The consequences of importing when you didn't want to are also going to be much higher than not importing when you wished you had.

Some examples of cases where it's harmful to inline remotes are:
- Google fonts #632
- MyFonts.com (uses an imported CSS file to count font usage, not doing this is likely a license violation)

At the very least, it might be worth handling protocol-relative URLs differently.

**Comment by joseluisq** [comment]
:+1:

**Comment by joseluisq** [comment]
@wagenet I agree :+1: 

In fact, by default occurs something like this:
```sh
$ cleancss \
assets/css/plugins.css \
assets/css/style.css \
-do assets/css/bundle.css

Inlining local stylesheet: /home/joseluis/assets/css/plugins.css
Inlining local stylesheet: /home/joseluis/assets/css/style.css
Inlining remote stylesheet: http://fonts.googleapis.com/css?family=Dosis
Inlining remote stylesheet: http://fonts.googleapis.com/css?family=Libre+Baskerville:400italic
Inlining remote stylesheet: http://fonts.googleapis.com/css?family=Open+Sans:400,400italic,700
Original: 319522 bytes
Minified: 281374 bytes
Efficiency: 11.93%
Time spent: 921ms
ERROR: Broken @import declaration of "http://fonts.googleapis.com/css?family=Open+Sans:400,400italic,700" - timeout
```

**Comment by jakubpawlowicz** [comment]
Thanks @wagenet @joseluisq - that's worth considering at least for remote imports.

**Comment by wagenet** [comment]
What if we just don't do imports for `//` urls, i.e. we don't import from `//fonts.google.com`, but we would still import from `http://fonts.google.com`? The lack of a protocol implies that it's context sensitive whereas pre-importing stuff is clearly _not_ context sensitive.

**Comment by jakubpawlowicz** [comment]
@wagenet @joseluisq thanks for valuable input - both features (remote imports off, protocol-less imports off) landed on master, and are due in 4.0.

**Comment by joseluisq** [comment]
> @wagenet @joseluisq thanks for valuable input - both features (remote imports off, protocol-less imports off) landed on master, and are due in 4.0.

Awesome :+1:  !
